tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -128,6 +128,27 @@ public function theUserChangesTheDisplayNameOfUserToUsingTheGraphApi(string $byU
 		$this->featureContext->setResponse($response);
 	}
 
+	/**
+	 * @When /^the user "([^"]*)" changes the user name of user "([^"]*)" to "([^"]*)" using the Graph API$/
+	 * @When /^the user "([^"]*)" tries to change the user name of user "([^"]*)" to "([^"]*)" using the Graph API$/
+	 *
+	 * @param string $byUser
+	 * @param string $user
+	 * @param string $userName
+	 *
+	 * @return void
+	 * @throws GuzzleException
+	 * @throws Exception
+	 */
+	public function theUserChangesTheUserNameOfUserToUsingTheGraphApi(string $byUser, string $user, string $userName): void {
+		$response = $this->editUserUsingTheGraphApi($byUser, $user, $userName);
+		$this->featureContext->setResponse($response);
+		// need add user to list to delete him after test
+		if (!empty($userName)) {
+			$this->featureContext->addUserToCreatedUsersList($userName, '123');
```

Comment:
I think we can remember the correct password
```suggestion
			$this->featureContext->addUserToCreatedUsersList($userName, $this->getUserPassword($user));
```